src/main/java/org/openmicroscopy/ms/zarr/RequestHandlerForImage.java

```diff
@@ -360,24 +372,24 @@ public RequestHandlerForImage(Configuration configuration, PixelsService pixelsS
         this.patternForImageDir = Pattern.compile(path);
         this.patternForImageGroupDir = Pattern.compile(path + "(\\d+)/");
         this.patternForImageChunkDir = Pattern.compile(path + "(\\d+)/(\\d+([/.]\\d+)*)/");
-        this.patternForImageMasksDir = Pattern.compile(path + "masks/");
-        this.patternForMaskDir = Pattern.compile(path + "masks/(\\d+)/");
-        this.patternForMaskChunkDir = Pattern.compile(path + "masks/(\\d+)/(\\d+([/.]\\d+)*)/");
-        this.patternForMaskLabeledDir = Pattern.compile(path + "masks/labell?ed/");
-        this.patternForMaskLabeledChunkDir = Pattern.compile(path + "masks/labell?ed/(\\d+([/.]\\d+)*)/");
+        this.patternForImageMasksDir = Pattern.compile(path + LABEL_GROUP + "/");
+        this.patternForMaskDir = Pattern.compile(path + LABEL_GROUP + "/(\\d+)/");
+        this.patternForMaskChunkDir = Pattern.compile(path + LABEL_GROUP + "/(\\d+)/(\\d+([/.]\\d+)*)/");
+        this.patternForMaskLabeledDir = Pattern.compile(path + LABEL_GROUP + "/" + DEFAULT_LABEL_NAME + "/");
```

Comment:
Works okay in this case but among these may want to consider `\\Q`…`\\E` as in `Configuration.getRegexForNetPath` if there is any risk of a regex metacharacter someday appearing, e.g., if later allowing these to be configurable.  Not a blocker though.

Reply:
https://github.com/ome/omero-ms-zarr/issues/69